frontend/vite.config.js

```diff
@@ -7,6 +7,7 @@ export default defineConfig({
     proxy: {
       '/api': 'http://127.0.0.1:3000'
     },
-    port: 3030
+    port: 3030,
```

Comment:
FYI you can go in your package.json and add a script. If this is to "host" the website on an ip address, then there's also another way to do it. `	"scripts": {
		"dev": "vite dev",
		"build": "vite build",
		"preview": "vite preview",
		"lint": "prettier --plugin-search-dir . --check . && eslint .",
		"format": "prettier --plugin-search-dir . --write ."
	},` that's the package.json, you could add something like: `"host": "vite dev --host",` or change the scipt for "dev" to include the `--host` option. That's what we do here since it's basically an enhanced version of `npm run dev`.
	
But again, If what you did here already does that, then all good. Just an fyi of something I learnt.

P.S. you can lookup the exact syntax, but if you want to specify an IP, I think it's `"dev": "vite dev --host <your_ip_address>"`

Reply:
Good to know, thanks for the info! I will leave this part as is.